grakn-graph/src/main/java/ai/grakn/util/GraphLoader.java

```diff
@@ -72,22 +80,53 @@ public static GraphLoader preLoad(Consumer<GraknGraph> build){
         return new GraphLoader(build);
     }
 
+    public static GraphLoader preLoad(String [] files){
+        return new GraphLoader(null, files);
```

Comment:
Rather than a `null` argument and all the logic and checks surrounding it, you could pass in a lambda here that invokes `loadFromFile` with the array of files.